google/cloud/aiplatform/jobs.py

```diff
@@ -1046,6 +1101,26 @@ def network(self) -> Optional[str]:
         self._assert_gca_resource_is_available()
         return self._gca_resource.job_spec.network
 
+    def _get_web_access_uris(self) -> Dict[str, str]:
+        """Helper method to get the web access uris of the custom job
+
+        Returns:
+            (Dict[str, str]):
+                Web access uris of the custom job.
+        """
+        return dict(self._gca_resource.web_access_uris)
+
+    def _log_web_access_uris(self):
+        """Helper method to log the web access uris of the custom job"""
+
+        for worker, uri in self.web_access_uris.items():
```

Comment:
```suggestion
        for worker, uri in self._get_web_access_uris.items():
```
To avoid the additional sync call.